packages/components/src/components/radio-group/radio-group.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { ChangeEvent } from 'react';
+import classNames from 'classnames';
+import Text from '../text';
+
+type TItem = React.HTMLAttributes<HTMLDivElement> & { value: string; label: string; disabled: boolean };
```

Comment:
@yauheni-kryzhyk-deriv I didn't understand the reason for `& { value: string; label: string; disabled: boolean }` 
would you please explain why we need them?

Reply:
sure,
previously in Item component there were added like rest props to div, but div hasn’t such properties. So here we are describing this properties explicitly.
then we take them from props inside radiogroup, where children are Item components

Reply:
yes, agree. It looks weird.
Frankly speaking, I didn’t find better solution. Because here these values we pass inside rest props of div.